doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,71 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`_, it is possible to convert the app to an executable by taking a few precautions.
+Because it is necessary to have the VERSION of PyMAPDL for the app to work, the file need to be added next to our executable.
+
+Start by generating the `specification file <https://pyinstaller.org/en/stable/spec-files.html>`_ for the app.
+
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
+
+PyInstaller provides two modes for generating executables:
+
+- ``onedir`` (default): This mode generates a folder that includes the executable file along with its dependencies.
+- ``onefile``: In this mode, PyInstaller generates a single executable file.
+
+To generate the executable in "onefile" mode, include the argument ``--onefile`` in the command.
```

Comment:
```suggestion
To generate the executable in ``onefile`` mode, include the argument ``--onefile`` in the command:
```
The Vale error will go away if ``onefile`` is tagged as a literal to indicate it's a code, file path, or filename reference.